### ISSUE TITLE
Problems while training yolov3-tiny occlusion track?

### ISSUE THREAD

**Issue by jacklin602** [AlexeyAB/darknet#3081]
@AlexeyAB hi,

When I was training yolov3-tiny occlusion track, I got the strange loss chart.
![chart](https://user-images.githubusercontent.com/49774448/57008284-8b66c700-6c21-11e9-91ed-2f910980f430.png)
I've got no idea about that. Is there something wrong with my cfg file or my training data?

Here are some files below for training,
train txt file: [https://github.com/jacklin602/training_file/blob/master/train.txt](url)
Training data includes two video files. I collected several frame sequences from each video.

cfg file: [https://github.com/jacklin602/training_file/blob/master/yolov3-tiny_occlusion_track.cfg](url)

**Comment by AlexeyAB** [comment]
@jacklin602 Hi,

* How many images are in your training dataset?
* You must train your model with `batch=8 subdivisions=4` in cfg-file or higher

**Comment by jacklin602** [comment]
@AlexeyAB 

>     * How many images are in your training dataset?
Currently, I just have 449 images in my training dataset. Would it be ok to have fewer frames in each frame sequences?
My goal is to detect five classes of vehicle, including bus, truck, sedan, motorcycle and trailer.
However, the numbers of sedan and motorcycle are much higher than other three classes.
Is this one of the reason that cause the loss chart wierd?

>     * You must train your model with `batch=8 subdivisions=4` in cfg-file or higher
Ok, I'll try it. But can I ask the reason why?
Thank you

**Comment by AlexeyAB** [comment]
Because higher `batch` reduces fluctuation.

Each sequence should have at least 100 frames.
